content/en/docs/setup/install/multicluster/shared-gateways/index.md

```diff
@@ -169,7 +169,12 @@ This will be used to access pilot on `cluster1` securely using the ingress gatew
     This command sets the value to the gateway's public IP and displays it.
 
     {{< warning >}}
-    The command fails if the load balancer configuration doesn't include an IP address. The implementation of DNS name support is pending.
+    The command fails if the load balancer configuration doesn't include an IP address. The implementation of DNS name support is pending. In this case, you can use the following command:
+
+    {{< text bash >}}
+    $ export LOCAL_GW_ADDR=$INGRESS_HOST && echo ${LOCAL_GW_ADDR}
```

Comment:
Not sure if this should also consist of `port`, I saw `istiod` log and it throws error saying not able to separate address, `port` is missing. If I add `$SECURE_INGRESS_PORT` the installation show incomplete but all the pods including `istiod` and `ingressgateway` were in running state.  Please confirm.

```
$ export LOCAL_GW_ADDR=$INGRESS_HOST:$SECURE_INGRESS_PORT && echo ${LOCAL_GW_ADDR} 
```